sample-hacep-project/sample-hacep-project-client/src/main/java/org/kie/hacep/sample/client/ClientProducerDemo.java

```diff
@@ -49,10 +49,10 @@ private static Properties getProperties() {
         Properties props = CommonConfig.getStatic();
         props.put("bootstrap.servers", "my-cluster-kafka-bootstrap-my-kafka-project.<ip>.nip.io:443");
         props.put("security.protocol", "SSL");
-        props.put("ssl.keystore.location", "/<path>/openshift-drools-hacep/client/src/main/resources/keystore.jks");
-        props.put("ssl.keystore.password", "password");
-        props.put("ssl.truststore.location", "/<path>/openshift-drools-hacep/client/src/main/resources/keystore.jks");
-        props.put("ssl.truststore.password", "password");
+        props.put("ssl.keystore.location", "/<path>/openshift-drools-hacep/sample-hacep-project/sample-hacep-project-client/src/main/resources/keystore.jks");
```

Comment:
Are `ssl.keystore.location` and `ssl.truststore.location` always the same? If yes please use a single property and just duplicate the keys so user can specify it only once. Same comment for `ssl.keystore.password` and `ssl.truststore.password`

Reply:
The password values can't be duplicated because one is the password for the keystore and one for the key